climakitae/explore/vulnerability.py

```diff
@@ -429,19 +447,23 @@ def cava_data(
             selections = DataParameters()
             selections.data_type = "Gridded"
             selections.downscaling_method = downscaling_method
-            selections.scenario_historical = [historical_data]
-            selections.scenario_ssp = clean_params["ssp_selected"]
+            selections.scenario_ssp = (
+                clean_params["ssp_selected"]
+                if historical_data != "Historical Reconstruction"
+                else []
+            )
             selections.timescale = "hourly"
             selections.variable_type = clean_params["variable_type"]
             selections.variable = variable
             selections.latitude = (
                 lat - 0.02,
                 lat + 0.02,
-            )  # Q: What padding to put for lat/lon values?
+            )
             selections.longitude = (lon - 0.02, lon + 0.02)
-            selections.time_slice = (time_start_year, time_end_year)
             selections.resolution = "3 km"
             selections.units = units
+            selections.scenario_historical = [historical_data]
```

Comment:
This is important to have moved! Otherwise, these two parameters get override to default parameters. I think this is a _**bug**_.

Reply:
@claalmve can you provide more context on this? Is this a bug? Do we need to fix it for climakitae as a whole or is this pilot specific?

Reply:
I think this is a bug, since it requires the selections to be defined in a particular order (which is hard to assume a user would input them in a particular order). I know some of this is intentional logic (i.e. if you choose Statistical data, selections doesn't allow you to select `Air Temperature at 2m`), but there could be better error handling so that users don't get confused by the need to order the selections parameters a particular way.

This would be a bug for climakitae as a whole.